tools/scylla-sstable.cc

```diff
@@ -2616,6 +2615,22 @@ void shard_of_operation(schema_ptr, reader_permit,
     writer.EndStream();
 }
 
+void shard_of_with_tablets() {
+    // TODO: support tablets
+    fmt::print(std::cerr, "tablets not supported");
+}
+
+void shard_of_operation(schema_ptr, reader_permit,
+                        const std::vector<sstables::shared_sstable>& sstables,
+                        sstables::sstables_manager& sstable_manager,
+                        const bpo::variables_map& vm) {
+    if (sstable_manager.config().enable_tablets()) {
```

Comment:
This is not reliable. It is possible to run the tool on a machine where the server's configuration is not available, so this will just return the default value for this config.

Reply:
ack. added another option `--vnodes` allowing user to use `vnodes`.

Reply:
I think we should always require the user to provide the replication option, and not have a default at all. This way there are no surprises at all.

Reply:
ack.